UPGRADE.md

```diff
@@ -1,3 +1,26 @@
+# Upgrade to 2.7
+
+## Doctrine\DBAL\Platforms\AbstractPlatform::DATE_INTERVAL_UNIT_* constants deprecated
```

Comment:
@photodude looks like the rebase didn't go smooth, there are still some irrelevant changes. Could you please clean things up?

Reply:
ugh... I have no Idea how the following broke this
```sh
git fetch upstream
git rebase upstream/master
git push
```
I don't have experience fixing this kind of issue with a rebase that's gone wrong. 
About the only thing I know how to do at this point is to make a new PR.
(I'm willing to learn if there is another way without creating a new PR)

Reply:
You don't have to make a new PR. As soon as you have the changes in the proper state in your local repository, you can force-push them here. In order to identify the state before the rebase, you have a few options:

1. Use `git reflog` and identify the commit before the rebase and reset to it.
2. Alternatively, look at https://api.github.com/repos/photodude/dbal/events and try to identify that state there.

Then you reset to that state (`git reset --hard <sha>`) and push it here.

Right now, looks like (probably because of the merge commit) your branch contains multiple commits which create the `appveyor.yml` file which conflict between each other:
```
↪ git log --pretty=oneline | grep 'Create .appveyor.yml'
141e5a44fd66ef45f378a85ce99c13ddc8d4c021 Create .appveyor.yml
69ad228bf0c4910c76f20c32879c0faeb82a68ae Create .appveyor.yml
5606cc7cd384be4836f3c81856486fd9a43f41de Create .appveyor.yml
bd7539057b2229f2328f24b92f1cb65dc4f93bb5 Create .appveyor.yml
```
Therefore, without knowing what the end result is expected to look like, I don't know how to resolve these conflicts properly.

Currently, your pull request contains 851 which is a bit too much. You can estimate the size using `git log upstream/master.. --pretty=format:'' | wc -l`. If you reset to a different state, make sure it contains a reasonable number of commits before pushing.

Reply:
I `git reset --hard <sha>` to an earlier commit that I think was before that last rebase. 
then I squashed all of the commits with a `git reset upstream/master` so we don't have the 200+ commits. 

I hope all of that was correct to get this back to a reasonable state without the cruft that occurred in the last rebase.